src/discussions/posts/post-actions-bar/PostActionsBar.jsx

```diff
@@ -67,15 +67,15 @@ const PostActionsBar = () => {
       )}
       {enableInContextSidebar && (
         <>
-          <div className="border-right border-light-300 mr-3 ml-1.5 my-10px" />
-          <div className="justify-content-center mt-2.5 mx-3px">
+          <div className="border-right border-light-300 mr-2 my-10px" />
+          <div className="justify-content-center my-2">
             <IconButton
               src={Close}
               iconAs={Icon}
               onClick={handleCloseInContext}
               alt={intl.formatMessage(messages.close)}
               iconClassNames="spinner-dimensions"
-              className="spinner-dimensions"
+              className="spinner-dimensions-36 text-primary-500"
```

Comment:
Do not need to explicitly define `text-primary-500`. IconButton component default variant is primary. The primary variant uses the same color as defined in Figma.